bootstrap/kubeadm/controllers/kubeadmconfig_controller.go

```diff
@@ -58,6 +58,7 @@ type InitLocker interface {
 // +kubebuilder:rbac:groups=bootstrap.cluster.x-k8s.io,resources=kubeadmconfigs;kubeadmconfigs/status,verbs=get;list;watch;create;update;patch;delete
 // +kubebuilder:rbac:groups=cluster.x-k8s.io,resources=clusters;clusters/status;machines;machines/status,verbs=get;list;watch
 // +kubebuilder:rbac:groups="",resources=secrets;events;configmaps,verbs=get;list;watch;create;update;patch;delete
+// +kubebuilder:rbac:groups=cluster.x-k8s.io,resources=machinesets;machinedeployments;machinepools,verbs=get;list;watch
```

Comment:
Is this correct? Do we expect that the KubeadmConfig controller would need to get MachineSets or MachineDeployments?

Reply:
Yeah it should only be looking at machines and machinepools afaik

Reply:
mm i think there might be a bug in the tilt stuff that's copying the wrong manager into the docker image